drivers/pcie/host/pcie.c

```diff
@@ -370,6 +371,106 @@ pcie_bdf_t pcie_bdf_lookup(pcie_id_t id)
 	return PCIE_BDF_NONE;
 }
 
+static bool scan_flag(const struct pcie_scan_opt *opt, uint32_t flag)
```

Comment:
should it be inlined? (though the compiler will probably be smart by itself here)

Reply:
Yeah, I think the sane approach is to trust the compiler, and only if you find it not doing the right thing to override it manually with an inline declaration in the code. In this case I'm almost sure that it'll always be inlined, unless you explicitly disable optimizations.